integrated_channels/integrated_channel/exporters/learner_data.py

```diff
@@ -114,57 +119,81 @@ def export(self, **kwargs):
                 enterprise_customer_user__user_id=learner_to_transmit.id,
             )
             exporting_single_learner = True
-            LOGGER.info('[Integrated Channel] Exporting single learner.'
-                        ' Course: {course_run}, Enterprise: {enterprise_slug}, User: {user_id}'.format(
-                            course_run=course_run_id,
-                            enterprise_slug=self.enterprise_customer.slug,
-                            user_id=learner_to_transmit.id))
+            generate_formatted_log(
+                'Exporting single learner. Course: {course_run}, User: {user_id}'.format(
+                    course_run=course_run_id,
+                    user_id=learner_to_transmit.id
+                ),
+                channel_name=channel_name,
+                enterprise_customer_identifier=self.enterprise_customer.name
+            )
         enrollment_queryset = enrollment_queryset.order_by('course_id')
 
         # Fetch course details from the Course API, and cache between calls.
         course_details = None
 
-        LOGGER.info(
-            '[Integrated Channel] Beginning export of enrollments: {}'.format(list(enrollment_queryset.values()))
+        generate_formatted_log(
+            'Beginning export of enrollments: {enrollments}.'.format(
+                enrollments=list(enrollment_queryset.values()),
+            ),
+            channel_name=channel_name,
+            enterprise_customer_identifier=self.enterprise_customer.name
         )
 
         for enterprise_enrollment in enrollment_queryset:
 
```

Comment:
have you considered sticking a debug log here to basically say "are we even in this for loop lol"

Reply:
Though I guess the "Beginning..." log should suffice.

Reply:
Not a bad idea, looking for places where I can add more clarity.

Reply:
The `Beginning` log was supposed to accomplish that, yes.